From 6ac9c7f563fee0d31e7a750be3b23fe3fa899f8e Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Mon, 15 Jun 2020 16:54:56 +0100
Subject: [PATCH] Set CMAKE_INSTALL_LIBDIR to lib

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c733c6c..8eafbfa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,6 +8,7 @@ aux_source_directory(common COMMON_SRC)
 set(APRILTAG_SRCS apriltag.c apriltag_pose.c apriltag_quad_thresh.c)
 
 set(CMAKE_BUILD_TYPE Release)
+set(CMAKE_INSTALL_LIBDIR "lib") # lib prefix workaround for multi-arch
 
 # Library
 file(GLOB TAG_FILES ${CMAKE_SOURCE_DIR}/tag*.c)
-- 
2.17.1

