From de2b253d558f4e352524958e4ef8136965277dc1 Mon Sep 17 00:00:00 2001
From: Wolfgang Merkt <w.merkt@gmail.com>
Date: Mon, 15 Jun 2020 16:54:56 +0100
Subject: [PATCH] Set CMAKE_INSTALL_LIBDIR to lib

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8451bf4..edde563 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,6 +13,8 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
     set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS  "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif()
 
+set(CMAKE_INSTALL_LIBDIR "lib") # lib prefix workaround for multi-arch
+
 find_package(OpenCV QUIET)
 
 aux_source_directory(common COMMON_SRC)
-- 
2.25.1

